docs/03-dev-howtos/06-repressing.md

```diff
@@ -1,61 +1,105 @@
+
+- `FaciaMetaDataTest.should Include item list metadata`
+- `FaciaMetaDataTest.should Include organisation metadata `
+- `FaciaControllerTest.should render fronts in mf2 format` 
+- `FaciaControllerTest.should render fronts in mf2 format (no section provided)` 
+- `FaciaControllerTest.should render fronts with content that has been pre-fetched from facia-press` 
+
+This is because the pressed JSON does not match what `Facia` is expecting. [Read the Fronts architecture here](https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/docs/02-architecture/02-fronts-architecture.md).
+
+### Repressing fronts locally.
+
+#### Prerequisites
+
+* Check your `facia.stage` property in your `frontend.conf` file is pointing at DEV (*not* PROD!)
+* Check you are using your own switches file and not the prod or code one. 
+	* To do this, create your own properties file in the the S3 bucket named `aws-frontend-store`. Add it to your `frontend.conf` like this: `switches.key=DEV/config/switches-nbaltazar.properties`
+* Ensure you have the test AWS SQS queues in your `frontend.conf` properties
+	* Add two properties: `frontpress.sqs.tool_queue_url` and `frontpress.sqs.cron_queue_url`
+	* Set _both_ properties to the value `"https://sqs.eu-west-1.amazonaws.com/<id-here>/Frontend-TEST-page-presser-queue"`
+	* You can get the correct ID by going to SQS in the AWS console and finding the `Frontend-TEST-page-presser-queue` and find the URL field in the details tab
+* Ensure your CAPI endpoint is PROD (`content.api.host` in `frontend.conf`)
+
+#### Lets go.
+
+* Delete the files in `data/pressedPage/`
+* Run the `admin` project locally and turn on `facia-press-on-demand` *and* `facia-inline-embeds` [switch](http://localhost:9000/dev/switchboard#Facia)
+* Stop admin and run `facia-press` locally
+* Go to `http://localhost:9000` just to get the switches file warmed up, then
+	* curl -v -X POST http://localhost:9000/press/live/music
+	* curl -v -X POST http://localhost:9000/press/live/inline-embeds
+	* curl -v -X POST http://localhost:9000/press/live/uk
+	* curl -v -X POST http://localhost:9000/press/live/au/media
+* Stop facia press, switch to `facia` project and run `test`
+	* Re-running the tests will recreate the JSON in the `data/pressedPage` folder - you'll need to commit those files.
+
+
+## Shipping
+
 This is more tricky especially if you are making a breaking change (i.e. new facia can't work with the old format and old facia can't work with the new)
 
 ### Adding a new field
+
 If you just ship new facia without pressing things will break until the presser kicks in.
 
 #### Prepare the ground
-* choose a quiet time of day, and this could take an hour
-* if it's on, turn off auto deploy of preview (and CODE facia if you like)
-* goo deploy block
-* let the team know what you're doing
-* merge to master
+
+* Choose a quiet time of day, and this will take an hour
+* Can you `ssh` onto the facia-press instances for `CODE` and `PROD`?
```

Comment:
you could just link to a doc in platform that explains it and just keep this as one step